pkg/segment/context/context.go

```diff
@@ -41,6 +45,10 @@ const (
 	JBoss    = "jboss"
 )
 
+// By default, all values are anomized.
+// Add the (case-insensitive) preference parameter name here to have the corresponding value sent verbatim to telemetry.
+var clearTextPreferenceParams = []string{}
```

Comment:
Don't we want to send boolean parameters (UpdateNotification, ConsentTelemetry, Ephemeral) in clear form? Hash value will be very easily reversed, we might as well have the clear value.

For numeric values (*Timeout, RegistryCacheTime), I think we could also send numeric value, as there can be nothing nothing personal IMO.